compiler/rustc_error_messages/locales/en-US/parser.ftl

```diff
@@ -306,6 +306,7 @@ parser_expected_identifier = expected identifier
 parser_sugg_escape_to_use_as_identifier = escape `{$ident_name}` to use it as an identifier
 
 parser_sugg_remove_comma = remove this comma
+parser_sugg_misplaced_generic = place the generic parameter list after the function name: `{$generic_as_string}`
```

Comment:
This is leftover from my initial attempt of doing it via diagnostic struct derives